File: official/docs/ruby/orders/one_call_buy_order.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

order = EasyPost::Order.create(
  to_address: {
    id: 'adr_...'
  },
  from_address: {
    id: 'adr_...'
  },
  shipments: [
    { parcel: { weight: 10.2 } },
    { parcel: { predefined_package: 'FedExBox', weight: 17.5 } },
  ],  
  service: 'NextDayAir',
  carrier_accounts: ['ca_...'],
)

puts order

File: official/docs/python/orders/one_call_buy_order.py
import easypost


easypost.api_key = "EASYPOST_API_KEY"

order = easypost.Order.create(
    carrier_accounts=["ca_..."],
    service="NextDayAir",
    to_address={"id": "adr_..."},
    from_address={"id": "adr_..."},
    shipments=[
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 10.2,
            }
        },
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 17.5,
            }
        },
    ],
)

print(order)
